### DISCUSSION TITLE
Having an issue with SSR

### DISCUSSION THREAD

**Discussion by nibarulabs** [sveltejs/kit#9662] [General]
I'm building a sveltekit SPA into an existing api. I can't seem to figure out how to turn ssr off. Gone through the instructions and they're pretty straight forward.

In my src/routes/+layout.svelte I added

```
export const ssr = false;
```

but I get an error. This is my relevant vite startup:

```
5:25:55 PM [vite-plugin-svelte] ssr compile done.
package         files     time     avg
svelte-admin        1   32.1ms  32.1ms

src/routes/+layout.svelte
`export const ssr` will be ignored — move it to +layout(.server).js/ts instead. See https://kit.svelte.dev/docs/page-options for more information.
```

These are my package dependencies and versions:

```
    "@sveltejs/adapter-static": "^2.0.2",
    "@sveltejs/kit": "^1.5.0",
    "@tailwindcss/forms": "^0.5.3",
    "@tailwindcss/typography": "^0.5.9",
    "autoprefixer": "^10.4.14",
    "postcss": "^8.4.21",
    "svelte": "^3.54.0",
    "svelte-forms": "^2.3.1",
    "tailwindcss": "^3.3.1",
    "vite": "^4.2.0"
```    

I think I'm up to date and if anyone has any suggestions on what I could try it would be appreciated. Thanks.

**Comment by acoyfellow** [comment]
The error tells you the solution: `move it to +layout(.server).js/ts instead`, not the `+layout.svelte` file.

**Reply by nibarulabs** [reply]
Yah, I did that. Added a +layout.server.js in src/routes and still the same error.

My bad, I should have mentioned that it was in the server file. I just checked my layout file and it was commented out.

**Comment by nibarulabs** [comment]
Update: apparently by leaving the commented line

```
// export const ssr = false;
```

in my +layout.svelte was causing this warning/error. Not sure why, but once I removed that, deleted my .svelte-kit folder, and then re-ran, the message is gone.

**Comment by ouksal** [comment]
I ran into the same issue — it turned out the build was triggered just because of a comment in +page.svelte that mentioned "trailingSlash". Removing the comment fixed it.
Not sure why the build assumed I was setting trailingSlash = 'always' there.
